TimeManagement.py

```diff
@@ -0,0 +1,44 @@
+"""
+Time Management portion of the project
+"""
+from datetime import datetime
+from datetime import timedelta
+from datetime import datetime
+import os
+from playsound import playsound
+import time
+
+class TimeManagment():
+
+    def __init__(self):
+        self.time_start = datetime.now()
+        self.work_duration_hours = int(input("How many hours would you like to work?\n"))
+        self.work_duration_minutes = int(input("How many minutes would you like to work?\n"))

```

Comment:
```suggestion
        self.work_duration_minutes = int(input("How many minutes would you like to work?\n"))
```

Can this be condensed into one line? How many hours would you like to work insert (HH, MM)

Reply:
It could probably could but then I would have to change other things as well